cmd/kinesumer/cmd_shards.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"fmt"
+	"math/big"
+	"sort"
+	"strings"
+
+	"github.com/codegangsta/cli"
+	"github.com/remind101/kinesumer"
+)
+
+var cmdShards = cli.Command{
+	Name:    "shards",
+	Aliases: []string{"sh"},
+	Usage:   "Gets the shards of a stream",
+	Action:  runShards,
+	Flags:   flagsStream,
+}
+
+type nums []*big.Int
+
+func (n nums) Len() int {
+	return len(n)
+}
+
+func (n nums) Swap(i, j int) {
+	n[i], n[j] = n[j], n[i]
+}
+
+func (n nums) Less(i, j int) bool {
+	return n[i].Cmp(n[j]) < 0
+}
+
+func (n *nums) UniqSort() {
+	sort.Sort(n)
+
+	tmp := make(nums, 0)
```

Comment:
make-ing a slice requires an initial size and we don't know how many items there'll be after deduplication